### PR TITLE
EF-121: Sign EF releases

### PR DESCRIPTION
**Pull request by sanych-sun**

### CODE REVIEW

File: evergreen/evergreen.yml [hunk 1]
```diff
@@ -252,6 +267,7 @@ functions:
         env:
           PACKAGES_SOURCE: ${PACKAGES_SOURCE}
           PACKAGES_SOURCE_KEY: ${PACKAGES_SOURCE_KEY}
+          NUGET_SIGN_CERTIFICATE_FINGERPRINT: ${$NUGET_SIGN_CERTIFICATE_FINGERPRINT}
```
Comment (BorisDog) [inline evergreen/evergreen.yml:270]: `${$NUGET_SIGN_CERTIFICATE_FINGERPRINT}` correct ?

Comment (sanych-sun) [reply evergreen/evergreen.yml]: Good catch! Thank you!

File: evergreen/evergreen.yml [hunk 2]
```diff
@@ -240,6 +253,7 @@ functions:
           ${PREPARE_SHELL}
           DRIVER_VERSION=${DRIVER_VERSION} \
           MONGODB_URI="${MONGODB_URI}" \
+          MONGODB_URI="${MONGODB_URI}" \
```
Comment (BorisDog) [inline evergreen/evergreen.yml:256]: typo?

File: evergreen/evergreen.yml [hunk 3]
```diff
@@ -404,6 +421,19 @@ tasks:
         vars:
           PACKAGES_SOURCE: "https://api.nuget.org/v3/index.json"
           PACKAGES_SOURCE_KEY: ${nuget_api_key}
+          NUGET_SIGN_CERTIFICATE_FINGERPRINT: ${NUGET_SIGN_CERTIFICATE_FINGERPRINT}
+
+
```
Comment (BorisDog) [inline evergreen/evergreen.yml:426]: Eemove empty line.

File: evergreen/template_ssdlc_compliance_report.md [hunk 4]
```diff
@@ -56,4 +55,10 @@ Coverity static analysis report is available <a href="https://coverity.corp.mong
 
 ## Signature information
 
-Blocked on <https://jira.mongodb.org/browse/EF-121>.
+Packages are signed with certificate with fingerprint: ${NUGET_SIGN_CERTIFICATE_FINGERPRINT}.
+Signature can be validated by running ```dotnet nuget verify``` command.
+
+For example signature of ```Mongodb.Driver.${PACKAGE_VERSION}.nupkg``` package can be verified by running:
```
Comment (BorisDog) [inline evergreen/template_ssdlc_compliance_report.md:61]: Change to EF package.

**Review by BorisDog — CHANGES_REQUESTED** [review]
Few comments

File: evergreen/template_ssdlc_compliance_report.md (hunk 4 above)
Comment (sanych-sun) [reply evergreen/template_ssdlc_compliance_report.md:61]: Ups, thanks

**Review by BorisDog — APPROVED** [review]
LGTM

**Pull request merged**